mautic/mautic
# Force set SITE URL as host to route loader

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
I think, this one line change also resolve this  pull request https://github.com/mautic/mautic/pull/6206
If you change Mautic URL, but stay on previus URL of Mautic, then a embeded form script link use old Mautic URL. This PR set host forced by SITE URL from Configuration.

![image](https://user-images.githubusercontent.com/462477/47074735-4e923000-d1fb-11e8-8596-5ae9f6173131.png)


[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1.  Setup two different URL to same Mautic url
2.  Go to detail page of  landing page
3. Check Public Preview URL 
4. Then go to Configuration and change Site URL of Mautic
5. Go to detail page of  landing page
6. Check Public Preview URL.
7. Site URL configuration change don't apply to URL of public preview, but we expect new SITE URL

#### Steps to test this PR:
1.  Repeat all steps
2.  Public Preview URL. of landing page should have new SITE URL even If you navigate via old URL  of Mautic
